src/modules/fundingManager/bondingCurve/abstracts/RedeemingBondingCurveBase_v1.sol

```diff
@@ -266,14 +266,24 @@ abstract contract RedeemingBondingCurveBase_v1 is
         if (collateralRedeemAmount < _minAmountOut) {
             revert Module__BondingCurveBase__InsufficientOutputAmount();
         }
-        // Transfer tokens to receiver
-        collateralToken.safeTransfer(_receiver, collateralRedeemAmount);
+
+        // Use abstract function to distribute collateral tokens
+        _distributeCollateralToken(_receiver, collateralRedeemAmount);
+
         // Emit event
         emit TokensSold(
             _receiver, _depositAmount, collateralRedeemAmount, _msgSender()
         );
     }
 
+    /// @dev    Abstract function to distribute collateral tokens.
```

Comment:
Same comment as in the `BondingCurveBase_v1`.

Reply:
Done